google/cloud/security/inventory/pipelines/load_org_iam_policies_pipeline.py

```diff
@@ -16,64 +16,113 @@
+            self.RESOURCE_NAME, cycle_timestamp, configs, crm_client, dao)
+        self.parser = parser
+
+    def _transform(self, iam_policies_map):
+        """Yield an iterator of loadable iam policies.
+
+        Args:
+            iam_policies_map: An iterable of iam policies as per-project
+                dictionary.
+                Example: {'project_number': 11111,
+                          'iam_policy': policy}
+                https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Policy
+
+        Yields:
+            An iterable of loadable iam policies, as a per-org dictionary.
+        """
+        for iam_policy_map in iam_policies_map:
+            iam_policy = iam_policy_map['iam_policy']
+            bindings = iam_policy.get('bindings', [])
+            for binding in bindings:
+                members = binding.get('members', [])
+                for member in members:
+                    member_type, member_name, member_domain = (
+                        self.parser.parse_member_info(member))
+                    role = binding.get('role', '')
+                    if role.startswith('roles/'):
+                        role = role.replace('roles/', '')
+                    yield {'org_id': iam_policy_map['org_id'],
+                           'role': role,
+                           'member_type': member_type,
+                           'member_name': member_name,
+                           'member_domain': member_domain}
+
+    def _retrieve(self):
+        """Retrieve the org IAM policies from GCP.
+
+        Args:
+            None
+
+        Returns:
+            iam_policies_map: List of IAM policies as per-org dictionary.
+                Example: {org_id: org_id,
+                          iam_policy: iam_policy}
+                https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Policy
+        """
+        try:
+            # Flatten the iterator since we will use it twice, and it is faster
+            # than cloning to 2 iterators.
+            return list(self.api_client.get_org_iam_policies(
+                self.name, self.configs.get('organization_id')))
+        except api_errors.ApiExecutionError as e:
+            raise inventory_errors.LoadDataPipelineError(e)
+
+    def run(self):
+        """Runs the data pipeline.
+
+        Args:
+            None
+
+        Returns:
+            None
+
+        Raises:
+            LoadDataPipelineException: An error with loading data has occurred.
+        """
+        org_id = self.configs.get('organization_id')
+        # Check if the placeholder is replaced in the config/flag.
+        if org_id == '<organization id>':
+            raise inventory_errors.LoadDataPipelineError(
+                'No organization id is specified.')
+
+        iam_policies_map = self._retrieve()
+
+        loadable_iam_policies = self._transform(iam_policies_map)
+
+        self._load(self.name, loadable_iam_policies)
```

Comment:
Nit: I think it would be more readable here and elsewhere in the pipeline code to just use the class constant (self.RESOURCE_NAME) instead of the initialized in the base class attribute self.name, which is harder to trace back when reviewing the code. Particularly since you go on to use self.RAW_RESOURCE_NAME below.